Cannot find module 'jsverify'

Hey, guys. Seems like in latest minor version there's a bug in types. I get following error:

`node_modules/ramda-adjunct/src/index.d.ts(1,29): error TS2307: Cannot find module 'jsverify'.`

Can it be so that you forgot to add `jsverify` to dependencies?

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,5 +1,3 @@
-import { GeneratorFn } from 'jsverify';
-
 declare var RA: RamdaAdjunct.Static;
 
 declare namespace RamdaAdjunct {

